### PR TITLE
add version

### PR DESCRIPTION
**Pull request by xlgao-zju**
Signed-off-by: Xianglin Gao <xlgao@zju.edu.cn>

---
This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/frakti/12)

### CODE REVIEW

**Comment by xlgao-zju** [comment]
@feiskyer updated.

File: pkg/hyper/hyper.go [hunk 1]
```diff
@@ -42,7 +41,7 @@ const (
 
 // Runtime is the HyperContainer implementation of kubelet runtime API
 type Runtime struct {
-	client *Client
+	*Client
```
Comment (feiskyer) [inline pkg/hyper/hyper.go]: Could you also move `hyperContextTimeout` and `E_NOT_FOUND` to `client.go`?

File: pkg/hyper/hyper.go [hunk 2]
```diff
@@ -53,12 +52,18 @@ func NewHyperRuntime(hyperEndpoint string) (*Runtime, error) {
...
 	}
 
-	return &Runtime{client: hyperClient}, nil
+	return &Runtime{Client: hyperClient}, nil
 }
 
 // Version returns the runtime name, runtime version and runtime API version
 func (h *Runtime) Version() (string, string, string, error) {
-	return hyperRuntimeName, "", "", fmt.Errorf("Not implemented")
+	version, apiVersion, err := h.Client.Version()
```
Comment (feiskyer) [inline pkg/hyper/hyper.go]: h.Version()

File: pkg/hyper/hyper.go [hunk 3]
```diff
@@ -75,20 +80,10 @@ func (h *Runtime) StopPodSandbox(podSandboxID string) error {
...
-
-	cxt, cancel := getContextWithTimeout(hyperContextTimeout)
-	defer cancel()
-
-	response, err := h.client.client.PodRemove(cxt, &request)
-	if response.Code == E_NOT_FOUND {
-		return nil
-	}
-
+	err := h.Client.RemovePod(podSandboxID)
```
Comment (feiskyer) [inline pkg/hyper/hyper.go]: h.RemovePod(podSandboxID)

File: pkg/hyper/hyper.go (hunk 2 above)
Comment (xlgao-zju) [inline pkg/hyper/hyper.go]: h.Version() will call Runtime.Version, so I rename Version to GetVersion in client.go

File: pkg/hyper/client.go [hunk 4]
```diff
@@ -42,4 +50,36 @@ func NewClient(server string, timeout time.Duration) (*Client, error) {
 	}, nil
 }
 
-// TODO: implements hyper client
+// Version gets hyperd version
```
Comment (feiskyer) [inline pkg/hyper/client.go]: Version -> GetVersion

**Comment by xlgao-zju** [comment]
@feiskyer updated

**Comment by xlgao-zju** [comment]
@feiskyer updated

**Comment by feiskyer** [comment]
LGTM. Thanks.

**Pull request merged**